### PR TITLE
Fix parse_avro docstring so that it can be shown in API doc

### PR DESCRIPTION
**Pull request by burgerkingeater**

### CODE REVIEW

File: tensorflow_io/core/python/experimental/parse_avro_ops.py [hunk 1]
```diff
@@ -32,11 +32,14 @@ def parse_avro(serialized, reader_schema, features, avro_names=None, name=None):
...
 
     :param serialized: The batched, serialized string tensors.
+
     :param reader_schema: The reader schema. Note, this MUST match the reader
                           schema from the avro_record_dataset. Otherwise,
                           this op will segfault!
+
     :param features: A map of feature names mapped to feature
                      information.
+
```
Comment (yongtang) [inline tensorflow_io/core/python/experimental/parse_avro_ops.py:42]: @burgerkingeater I think that may not work to fix the argument display. Can you follow the way other APIs exposing the arg in docstring, e.g., :
```
    Args:
    serialized: A vector (1-D Tensor) of strings, a batch of binary
      serialized `Example` protos.
    reader_schema: A scalar string representing the reader schema.
```

**Comment by yongtang** [comment]
@burgerkingeater Any update on this PR?

**Comment by burgerkingeater** [comment]
@yongtang thanks for the reminder, will get to it today.

**Comment by burgerkingeater** [comment]
@yongtang updated, thanks

File: tensorflow_io/core/python/experimental/parse_avro_ops.py (hunk 1 above)
Comment (burgerkingeater) [reply tensorflow_io/core/python/experimental/parse_avro_ops.py]: updated, thanks

**Pull request merged**